examples/all-clusters-app/nrfconnect/prj.conf

```diff
@@ -25,6 +25,13 @@ CONFIG_CHIP_PROJECT_CONFIG="main/include/CHIPProjectConfig.h"
 CONFIG_CHIP_DEVICE_PRODUCT_ID=32769
 CONFIG_STD_CPP17=y
 
+# Enable Matter pairing automatically on application start.
+CONFIG_CHIP_ENABLE_PAIRING_AUTOSTART=y
```

Comment:
It used to not be enabled on all platforms, so you're changing this intentionally, right?

Reply:
Yes, I just haven't seen the reason to not enable it, since it is enabled in other examples.